lib/api/actions/functions.php

```diff
@@ -536,76 +540,95 @@ function _beans_add_anonymous_action( $hook, $callback, $priority = 10, $args =
 
 }
 
+// @codingStandardsIgnoreEnd
+
 /**
  * Render action which can therefore be stored in a variable.
  *
- * @ignore
+ * @since 1.5.0
+ * @access private
```

Comment:
Currently the WP Parser used to automatically generate Getbeans site PHPDoc doesn't support the `@access` tag, only the `@ignore` which we can frequently find in WP core. That said, we can add support for it using the `wp_parser_pre_import_item` filter. @hellofromtonya could you perhaps create a pre-release tasks list and add this to the list?

Reply:
>That said, we can add support for it using the wp_parser_pre_import_item filter. @hellofromtonya could you perhaps create a pre-release tasks list and add this to the list?

Definitely.  Let me open an issue to remind us of this item.  

As we are updating the codebase, I think it's a good idea to mark private functions.  Typically, private functionality is not documented, as they should not be used within the theme or plugins.

Reply:
Absolutely agree, that said removing the `@ignore` tag removes the flag which currently tell the WP Parser not to add it to the site, hence the need to use the `wp_parser_pre_import_item` to tell the WP Parser to exclude anything with the `@access private` tag instead, otherwise these functions will be added to the site which we don't want.

Reply:
Done. DocBlocks for private functions now have this format:

```
 * @since 1.5.0
 * @ignore
 * @access private
```